### PR TITLE
Add Self-Review Questionnaire: Security and Privacy

### LINKED ISSUE #7: Add Self-Review Questionnaire: Security and Privacy
In order to file a request for a [TAG design-review](https://github.com/w3ctag/design-reviews/issues/new?assignees=&labels=Progress%3A+untriaged%2C+Review+type%3A+early+review&projects=&template=005-early-design-review.md&title=), it's best if the explainer includes [W3C's "Self-Review Questionnaire: Security and Privacy"](https://w3ctag.github.io/security-questionnaire/) (following @noamr's standard at his [`moveBefore` explainer](https://github.com/noamr/dom/blob/spm-explainer/moveBefore-explainer.md#self-review-questionnaire-security-and-privacy))

### PR DESCRIPTION
**Pull request by weizman**
closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +240,94 @@ This already goes with how CSP is currently enforcing its rules canonically in t
...
+
+No
+
+> 07.  Do the features in your specification expose information about the underlying platform to origins?
+
+No
+
+> 08.  Does this specification allow an origin to send data to the underlying platform?
+
+The answer is basically no aside for the value of the configured new CSP directive which should be parsed into a URL of a remote resource
```
Comment (yoavweiss) [inline README.md:276]: I think this can just be "no"

File: README.md [hunk 2]
```diff
@@ -239,6 +240,94 @@ This already goes with how CSP is currently enforcing its rules canonically in t
...
+
+No
+
+> 13.  What temporary identifiers do the features in this specification create or expose to the web?
+
+None
+
+> 14.  How does this specification distinguish between behavior in first-party and third-party contexts?
+
+This feature expects the website to provide a URL to a remote JavaScript file to be loaded within new same origin realms only, so naturally there's an important distinction where the file should only be loaded and executed within first-party contexts and not third-party contexts. Telling them apart should (hopefully) be rather straight forward since CSP is already very good at this exactly.
```
Comment (yoavweiss) [inline README.md:301]: IIUC, the script would only run in same-origin contexts (which is stricter than first-party/same-site contexts)

**Review by yoavweiss — COMMENTED** [review]
LGTM

**Review by yoavweiss — APPROVED** [review]
Approving % a couple of suggestions for slightly stricter language

File: README.md (hunk 1 above)
Comment (weizman) [reply README.md]: b05732f2dc2275112035657c77bf83ad86217969

File: README.md (hunk 2 above)
Comment (weizman) [reply README.md:301]: Of course, but to answer the question, isn't that a distinction we are expected to manifest in the specification? Isn't this question asking us to clarify how the 1st vs 3rd party distinction is relevant here?

Comment (yoavweiss) [reply README.md:301]: "This feature enables sites to run a remote JavaScript file in the context of new same-origin realms, and only such realms. As same-origin is stricter than same-site, the distinction between first-party and third-party contexts falls out of that restriction."?

Comment (weizman) [reply README.md:301]: Only now realizing how "party" refers to "site"s as opposed to "origin"s - on it

Comment (weizman) [reply README.md]: 67cdc362badc935f0c7928e8312804de476d9fd8

**Pull request merged**